### PR TITLE
Refactoring MediaInfo_Config_MediaInfo

### PR DESCRIPTION
**Pull request by pavel-pimenov**

### CODE REVIEW

File: Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp [hunk 1]
```diff
@@ -1788,8 +1789,7 @@ Ztring MediaInfo_Config_MediaInfo::File_Duplicate_Set (const Ztring &Value_In)
     }
     if (!ToReturn.empty())
         ToReturn.erase(ToReturn.begin()); //Remove first ";"
-
-    CS.Leave();
+    }
```
Comment (JeromeMartinez) [inline Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp]: I am not a big fan of adding {}, but also not against it if you prefer it + fix compilation issues.

File: Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp [hunk 2]
```diff
@@ -1800,7 +1800,7 @@ Ztring MediaInfo_Config_MediaInfo::File_Duplicate_Get (size_t AlreadyRead_Pos)
     CriticalSectionLocker CSL(CS);
     if (AlreadyRead_Pos>=File__Duplicate_List.size())
         return Ztring(); //Nothing or not more than the last time
-    Ztring Temp=File__Duplicate_List[AlreadyRead_Pos];
+    const Ztring Temp=File__Duplicate_List[AlreadyRead_Pos];
```
Comment (JeromeMartinez) [inline Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp]: why not return File__Duplicate_List[AlreadyRead_Pos]; here?
(I missed something?)

File: Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp (hunk 1 above)
Comment (pavel-pimenov) [inline Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp]: CriticalSectionLocker: exception-safe code. (RAII)

Comment (JeromeMartinez) [inline Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp]: ok. don't forget the compilation error due to this change. (AppVeyor is complaining)

**Pull request merged**